slow_tests/test_benchmark.py

```diff
@@ -47,7 +47,9 @@ def benchmark(
     num_batches=100,
     steps_per_batch=1000,
     compute_du_dp_interval=100,
-    compute_du_dl_interval=0
+    compute_du_dl_interval=0,
+    barostat=False,
+    barostat_interval=25,
```

Comment:
1) did we want to use barostat_interval = 0 as a signal for False?
2) what happens if we actually passed in a barostat interval of 0 to the custom_ops class, does actually do a mod zero of something?

Reply:
1) That would work for me

2) It faults, will fix it up to raise a more meaningful error